torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3985,13 +4015,16 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
     OpInfo('addr',
            dtypes=all_types_and_complex_and(torch.bool, torch.bfloat16, torch.float16),
            backward_dtypes=all_types_and_complex_and(torch.bool, torch.bfloat16),
-           backward_dtypesIfCUDA=all_types_and_complex_and(torch.bool),
+           backward_dtypesIfCUDA=all_types_and_complex_and(torch.bool, *[torch.bfloat16] if CUDA11OrLater else []),
```

Comment:
Let's make sure we ci-all this to avoid a revert with these nuanced CUDA support statements